en/config_fw/advanced_tuning_guide_fixedwing.md

```diff
@@ -79,9 +79,8 @@ Furthermore, these two values define the height rate limits commanded by the use
 
 ### L1 Controller Tuning (Position)
```

Comment:
Are they still referred to as "L1" ? I.e. should we rename FW Path Control Parameters?

Reply:
Ah sorry missed that, yes, your suggestion is better